app/services/public_transportation/import_service.rb

```diff
@@ -93,7 +94,13 @@ def call
     def parse_calendar_dates(zip_file)
       p "Parsing calendar_dates.txt"
       calendar_dates_counter = 0
+
+      # cleanup stop_times bevor import
+      @redis.delete_all_gtfs_calendar_dates(@data_provider_id)
```

Comment:
cleanup stop times and than calling deletion of calendar dates? what about `# cleanup before import` as comment without specifying?

Reply:
updated comments

Reply:
would you like to change the `bevor`s to `before`s? ;)

Reply:
thanks!